### PR TITLE
Fixing dimensions issue in ipol and add tests,

### PR DESCRIPTION
**Pull request by heistermann**
This refers to issue #171. Fixes the inconsistent behaviour in ipol in case flat arrays are passed. Also adds some tests for ipol as well as adjust.

### CODE REVIEW

File: wradlib/ipol.py [hunk 1]
```diff
@@ -200,6 +202,14 @@ def __call__(self, vals, maxdist=None):
         """
         Evaluate interpolator for values given at the source points.
 
+        You can interpolate multiple datasets of source values (`vals`) at
+        once: the `vals` array should have the shape (number of source points,
+        number of source datasets). If you want to interpolate only one set
+        of source values, `vals can have the shape (number of source points,
```
Comment (kmuehlbauer) [inline wradlib/ipol.py]: Backtick missing at `vals`

File: wradlib/ipol.py [hunk 2]
```diff
@@ -279,6 +289,14 @@ def __call__(self, vals):
...
```
Comment (kmuehlbauer) [inline wradlib/ipol.py]: same as above

File: wradlib/ipol.py [hunk 3]
```diff
@@ -365,6 +383,14 @@ def __call__(self, vals, fill_value=np.nan):
...
```
Comment (kmuehlbauer) [inline wradlib/ipol.py]: same here

File: wradlib/ipol.py [hunk 4]
```diff
@@ -642,6 +668,14 @@ def __call__(self, vals):
...
```
Comment (kmuehlbauer) [inline wradlib/ipol.py]: and here

File: wradlib/ipol.py [hunk 5]
```diff
@@ -797,6 +834,14 @@ def __call__(self, vals, src_drift=None, trg_drift=None):
...
```
Comment (kmuehlbauer) [inline wradlib/ipol.py]: here also

**Comment by kmuehlbauer** [comment]
@heistermann If we pull this in now, then it will be in the next release (0.10.2 or 0.11.0). Considering what you mentioned in #171, we wait until preparation of release 1.0.0, right?

**Comment by heistermann** [comment]
👍

**Comment by kmuehlbauer** [comment]
@heistermann I cherry-picked you changes and they are taken care of in PR #226. Closing.

**Pull request closed without merging**